julian-mainardis-globant/JavaBaseProject
# Julian.mainardis/unit testing

## Description

Add all the unit testings.

## Story Type

- New feature :white_check_mark:
- Change request
- Bug fix
- Tech

## How Can This Be Tested?

Run all the unit tests one by one or all together to verify that all pass without errors.

## Evidences

Coverage:
![image](https://user-images.githubusercontent.com/109156192/185201350-9a5f0d04-b20b-465a-9e1c-1e6e7c1eb918.png)
![image](https://user-images.githubusercontent.com/109156192/185201383-ea2d3123-0c14-4009-bc95-90ef883e0250.png)


## Checklist

- All unit Test passed :white_check_mark:

## Package Structure
<img width="294" alt="image" src="https://user-images.githubusercontent.com/109156192/185201708-416201f8-f713-4a85-b7ba-02b791e48abf.png">
